docs/wiki/ModulesDesign.md

```diff
@@ -185,10 +422,8 @@ Within a bicep file, follow the following conventions:
     "@
     $resourceRoles = @()
     $rawRolesArray = $rawRoles -split "`n"
-    for ($i = 0; $i -lt $rawRolesArray.Count; $i++) {
-      if($i % 5 -eq 0) {
-        $resourceRoles += $rawRolesArray[$i].Trim()
-      }
+    for ($i = 0; $i -lt $rawRolesArray.Count; $i+5) {
```

Comment:
Move this to a tool/script thing and reference in the "RBAC" section of the patterns?

Reply:
Done